File: fibonacci.r
a=0
b=1
ans=0
n<-readline(prompt="enter total desired values for fibonacci series : ")
n<-as.integer(n)
if(n==1){
  cat(a)
}else if(n==2){
  cat(a," ",b)
}else{
  cat(a," ",b," ")
  for(i in 2:n)
  {
    ans=a+b
    a=b
    b=ans
    cat(ans," ")
  }
}

File: fibonacci.jl
print("enter the no. of terms for fibonacci series : ")
n = readline()
n = parse(Int, n)
a = 0
b = 1
println("fibonacci series is : ")
if (n==1)
    println(a, " ")
elseif (n==2)
    println(a, " ", b, " ")
else
    print(a, " ", b, " ")
    for i in 1:n-2
        c = a + b
        print(c, " ")
        global a = b
        global b = c
    end
    println()
end